pages/historicaldocuments/frus-history/index.xml

```diff
@@ -5,19 +5,8 @@
         data-template-at="content">
       <div>
          <div class="row">
-            <div class="hsg-breadcrumb-wrapper">
-               <ol class="breadcrumb" data-template="app:fix-links">
-                  <li>
-                     <a href="$app">Home</a>
-                  </li>
-                  <li>
-                     <a href="$app/historicaldocuments">Historical Documents</a>
-                  </li>
-                  <li>
-                     <a href="$app/historicaldocuments/frus-history">History of the <em>Foreign
-                                Relations</em> Series</a>
-                  </li>
-               </ol>
+            <div data-template="pages:breadcrumb">
+               <!--<p id="breadcrumb-title">History of the <em>Foreign Relations</em> Series</p>-->
```

Comment:
I noticed, that this breadcrumb title has changed for `historicaldocuments/frus-history`: Instead of former "History of the <em>Foreign Relations</em> Series", the title "Toward “Thorough, Accurate, and Reliable”: A History of the Foreign Relations of the United States Series" (`frus-history:monograph-title`) is displayed, but without the font-styling `italic`. Is this intended?